src/Route.jsx

```diff
@@ -11,15 +11,24 @@ const ICON_MAPPING = {
   },
 };
 
-export function getStopMarkersLayer(route, getStopInfo) {
+/* changes color of selected stop along a route */
+function selectedStopColor(stop, selectedStops) {
+  const selectedStopsIds
+  = selectedStops.map(currentstop => currentstop.sid);
+  if (selectedStopsIds.indexOf(stop.sid) !== -1) {
+    return [0, 255, 0];
+  }
+  return [255, 0, 0];
```

Comment:
@EddyIonescu how should we handle this? Right now I hardcoded the color to be green (default color for stops is red) if it's selected.
Should we make the color a param?
Should we add the option of changing the icon? Or is changing the color enough?

Reply:
Could we use `includes` here? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes